fundingmanager.go

```diff
@@ -3307,6 +3312,58 @@ func (f *fundingManager) pruneZombieReservations() {
 	}
 }
 
+// failPausedReservations informs the remote peer that we cannot continue a
+// previously started funding flow because we are shutting down. We only do this
+// for reservations where we artificially stalled/paused the progress because
+// the user needs to perform manual funding steps (like signing a PSBT for
+// example).
+func (f *fundingManager) failPausedReservations() {
```

Comment:
This looks fine to me, but I'm wondering if it's necessary? I would imagine that the responding node would just forget of the incomplete reservation eventually. Did you run into any issues while testing that required this?

Reply:
Yes, I noticed that if I shut down `lnd` while a PSBT funding was in progress (to test that no lingering channel ready somewhere block shutdown) I noticed that you then have to wait 10 minutes until you can try opening a channel to the same node as before again, otherwise you would run into "maximum number of pending channels exceeded" error. But it's quite unlikely this happens, I agree. I'm fine with removing it again.

Reply:
Ah, you are right. I tested it again and there was no wait. Must have been something else I didn't clean up in my first tests.
I looked into this again and tried canceling all reservations. But at the moment of the shutdown of the funding manager, the wallet has already stopped its message consuming loop and we get a deadlock.
And since we know that the reservation and coins locks are in-memory only on our side it's probably OK to do nothing on shutdown, as it was before.